packages/manager/modules/pci/src/projects/project/storages/object-storage/object-storage.html

```diff
@@ -0,0 +1,38 @@
+<div data-ui-view>
+    <oui-header
+        class="mb-3"
+        data-heading="{{:: 'pci_projects_project_storages_containers_title' | translate }}"
+        data-description="{{::$ctrl.project.projectId }}"
+    >
+        <div data-ovh-alert="{{ $ctrl.alerts.dashboard }}"></div>
```

Comment:
This directive come from `@ovh-ux/ng-ovh-utils` module which is:
- Deprecated
- Not defined as a dependency (nor peerDepencendy) of this following `@ovh-ux/manager-pci` module

**question:** Why no prefer using [`<oui-message>`](https://ovh.github.io/ovh-ui-kit/?path=/story/design-system-components-message-webcomponent--default) component here?